dataprofiler/profilers/profile_builder.py

```diff
@@ -329,6 +329,23 @@ def __init__(self, data, samples_per_update=None, min_true_samples=None,
         elif not isinstance(profiler_options, ProfilerOptions):
             raise ValueError("The profile options must be passed as a "
                              "ProfileOptions object.")
+
+        try:
```

Comment:
I'm not sure turning off the data labeler in general is appropriate as a labeler maybe made without TF. Need to think more on this of how best to address.

Reply:
I actually don't know where else to put this, but currently there's no other reasonable model besides tensorflow. 

I agree this is very convoluted, but I don't know where else to put this.

Reply:
I agree with Jeremy, a labeler could be used without tensorflow (like our own regex model). Going to look in the repo for where I would put this.